src/sql/parts/registeredServer/viewlet/serverTreeRenderer.ts

```diff
@@ -146,9 +147,10 @@ export class ServerTreeRenderer implements IRenderer {
 			}
 		}
 
-		let databaseName = connection.databaseName ? connection.databaseName : '<default>';
-		let userName = connection.userName ? connection.userName : "Windows Authentication";
-		let label = connection.serverName + ', ' + databaseName + ' (' + userName + ')';
+		let label = connection.title;
+		if (!connection.isConnectionOptionsValid) {
+			label = localize('loading', 'Loading ...');
```

Comment:
I think this should be "Loading..." (without the space) to be consistent with other parts of the code